api/src/utils/user-utils.ts

```diff
@@ -0,0 +1,58 @@
+import { difference } from 'lodash';
+import { isAdmin } from './auth-utils';
+import User from '../models/user';
+import { SessionUser } from './helpers';
+
+// All fields in user
+const allFields = Object.keys(User.schema.paths);
+
+const nonViewableFields = ['oauthID', '__v'];
+
+const nonEditableFields = ['_id', 'oauthID', '__v'];
+// Fields that non-Admins and other users cannot view
+const privateFields = ['phone'];
+// Fields that non-Admins cannot edit
+const adminEditOnlyFields = ['currentTeams', 'role', 'email', 'races'];
```

Comment:
How about `adminEditableFields`?

Reply:
Now thinking about it, this name wouldn't explicitly denote that other users can't edit these fields, but I think it can be inferred - what are your thoughts on this?

Reply:
I think the special specification of something being admin viewable intuitively states that this is admin exclusive. I think trying to specify this in the variable will make it messy, so I might add a comment to explicitly state this